client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -143,26 +144,43 @@ const VerticalSelect: FunctionComponent< StepProps > = ( {
 		<Question
 			label={ label }
 			displayValue={ displayValue }
-			isActive={ isActive }
+			isActive={ isActive || ! displayValue }
 			onExpand={ onExpand }
 		>
 			<div className="vertical-select">
+				{ ! inputValue && (
+					<AnimatedPlaceholder
+						texts={ [
+							NO__( 'football' ),
+							NO__( 'shopping' ),
+							NO__( 'cars' ),
+							NO__( 'design' ),
+							NO__( 'travel' ),
```

Comment:
We'll need to add context here so that translators know what they're translating: 

```
_x( 'football', 'Input value example to continue sentence "My site is about ..."' )
```